src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+  --shadow-hover-focus: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+  --linear-hover-focus: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.form__loop {
+  background-image: url("./images/Search.svg");
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.form {
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  outline: none;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  color: #3d4e61;
+}
+
+.form-big {
+  position: relative;
+  display: flex;
+  align-items: center;
+  margin: 20px 0;
+}
+
+.form__loop--size_big {
+  width: 19px;
+  height: 19px;
+}
+
+.form-big__big-loop {
+  position: absolute;
+  left: 26px;
+}
+
+.form--input_big {
+  height: 28px;
+  font-size: 16px;
+}
+
+.form-big__form {
+  padding: 20px;
+  padding-left: 62px;
+}
+
+.form--input_big:hover {
+  box-shadow: var(--shadow-hover-focus);
+}
+
+.form--input_big:focus {
+  border-radius: 3px;
+  background: var(--linear-hover-focus);
+  box-shadow: var(--shadow-hover-focus);
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: bold;
+}
+
+.form-small {
+  position: relative;
+  display: flex;
+  align-items: center;
+}
```

Comment:
you're duplicating code for form-small and form-big.
You cand add default styles for form and override some of them for small one.
for example:

```css
.form {
  // some common styles goes here, like display flex, etc.

padding: // your value for big one.  it will be default value
}

.form-small {
padding: // some values which is different from default.
```

Reply:
check this everywhere